### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by philamwild**

### CODE REVIEW

File: frontend/src/App.scss [hunk 1]
```diff
@@ -1 +1,4 @@
-
+* {
+  margin: 0;
+  padding: 0;
```
Comment (TheoBoschetti) [inline frontend/src/App.scss]: is it not already in reset.css ?

File: frontend/src/components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import "./styles/Navbar.scss";
+
+function Navbar() {
+  const [menuIsOpen, setOpen] = useState(false);
```
Comment (TheoBoschetti) [inline frontend/src/components/Navbar.jsx]: [IsMenuopen, setIsMenuOpen]

File: frontend/src/components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            </NavLink>
+            <NavLink activeClassName="active" to="/blog">
+              Blog
+            </NavLink>
+          </div>
+        </div>
+
+        <div className="mobileNavbar">
+          <p>tttt</p>
+          <h1>TETATETE</h1>
```
Comment (TheoBoschetti) [inline frontend/src/components/Navbar.jsx]: We already have a h1 markup in component tagline

File: frontend/src/components/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+              Causes soutenues
+            </NavLink>
+            <NavLink activeClassName="active" to="/blog">
+              Blog
+            </NavLink>
+          </div>
+        </div>
+
+        <div className="mobileNavbar">
+          <p>tttt</p>
```
Comment (TheoBoschetti) [inline frontend/src/components/Navbar.jsx:34]: It will be a img here imo

**Review by TheoBoschetti — CHANGES_REQUESTED** [review]

File: frontend/src/components/Navbar.jsx (hunk 3 above)
Comment (LoicBrassart) [reply frontend/src/components/Navbar.jsx]: More than that, please remember to not write the final website name in your code anymore

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request closed without merging**